js/blocks/components/repeater-rows.js

```diff
@@ -109,13 +118,15 @@ import { Fields } from './';
 			rowRefTo.classList.add( 'row-to' );
 			rowRefFrom.classList.add( 'row-from' );
 
-			setTimeout( () => {
-				rowRefTo.classList.remove( 'row-to' );
-				rowRefFrom.classList.remove( 'row-from' );
-			}, 1000 )
+			this.timeouts.push(
```

Comment:
This stores the timeout IDs in `this.timeouts`, and clears them in `componentWillUnmount()`